storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -916,10 +911,7 @@ bool copy_redo_encryption_info() {
                           log_buf.get() + encryption_offset, encryption_offset,
                           Encryption::INFO_SIZE);
   if (!success) {
-    msg_ts(
-        "xtrabackup: Fatal error: cannot write encryption to redo log "
-        "%s.\n",
-        dst_path);
+    xb::error() << "cannot write encryption to redo log " << dst_path;
```

Comment:
I'm not aware of the context here. From what it reads,this is a fatal error. Can we not use `xb::fatal()` here?

Reply:
Currently, I am avoiding fatal() because it creates core file.

Reply:
@venkatesh-prasad-v it seems there is a way to avoid the assert and just get the fatal category. I will investigate and fix it